### ISSUE TITLE
Monetary currency representations

### ISSUE THREAD

**Issue by elf-pavlik** [elf-pavlik/currency#2]
In our use cases, so far I see two distinct representation of monetary currencies
- **Cash** as discussed in #1 driven by [Dental Care](https://github.com/valueflows/valueflows/blob/master/use-cases/dental-care.md) use case
- **BankCredits** where we could possibly use [Currency exchange (ISO4217)](https://github.com/valueflows/valueflows/blob/master/use-cases/currency-exchange-iso4217.md)

Does anyone see another distinct representation of monetary currencies than those two?

I can imagine that agents would like to query their datasets for 'how much MXN i have rights to' and get aggregation amounts based on state of all their cash stocks and bank accounts. Or get break down _Cash_, _BankCredits_, _Total_

**Comment by bhaugen** [comment]
Digital currencies like bitcoin, mutual credit currencies?

**Comment by elf-pavlik** [comment]
> mutual credit currencies

AFAIK mutual credit systems use a central bank to manage that particular mutual credit system, IMO those credits qualify as _BankCredit_, they need this bank to keep track of all the credits and act as authority for that particular mutual credit system,

Similar applies to hours in time bank, just that units have mapping to physical units.

> Digital currencies like bitcoin

They also seem a _BankCredit_, just we deal with distributed ledger used as a bank, rather than some centralized system.

If you see need to add more categories, let's do it and work on defining distinctions between them.

**Comment by bhaugen** [comment]
I don't think mutual credit systems use a bank, unless you mean metaphorically "something like a bank that is not really a bank". I could be wrong, just learning about them.

Re digital currencies, I am just getting my first real experience coding transactions sent to a blockchain, and they seem very different, and they also seem very centralized, which I could explain on request but it always gets me into arguments with bitcoin fans.

**Comment by elf-pavlik** [comment]
> I don't think mutual credit systems use a bank, unless you mean metaphorically "something like a bank that is not really a bank". I could be wrong, just learning about them.

For example instance of http://communityforge.net/en acts as a bank. AFAIK one can also configure it as time bank. If you find term Bank unfit, we could maybe use Authority & AuthorityCredits. Something to explain that you need an account with an certified authority for that monetary currency system and you prove that you control certain amount of that currency by saldo on that account managed by certified authority. AFAIK mutual credits systems use single central authority, just like time banks.

**Comment by bhaugen** [comment]
Yes, I would like to reserve the word Bank for banks. I would be ok with a metaphor, bank-like, or something else that describes all of those somewhat-like-a-bank situations in a more general way. I don't think Authority communicates the idea very well,  but I don't have a better word a the moment. Wordnet Search is my friend, will try that later if nobody has a better idea.

**Comment by elf-pavlik** [comment]
> I would like to reserve the word Bank for banks
- https://en.wikipedia.org/wiki/Bank
- https://en.wikipedia.org/wiki/Time-based_currency#Time_banks
- https://en.wikipedia.org/wiki/Mutual_credit

Maybe just _Credit_ or verbose _CreditOnAccount_. I think check/cheque also somehow fits here as payment method operating on those credits stored on accounts.
